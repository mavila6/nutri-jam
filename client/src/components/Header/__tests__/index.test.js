import React from "react";
// import { Link } from "react-router-dom";
import { render, cleanup } from "@testing-library/react";
// import { Navbar, Nav, Container, Modal, Tab } from "react-bootstrap";
import "@testing-library/jest-dom/extend-expect";
import AppNavBar from "..";

// const mockToggleHeader = jest.fn();
// const currentRecipe = {
//   title: "Caldo de Queso",
//   link: "caldos.com",
//   source: "Mi Abuela",
//   totalTime: 2,
//   ingredients: ["papas, queso, caldo, pablanos"],
// };

afterEach(cleanup);

describe("Header component", () => {
  //renders Header test
  // First Test
  it("renders", () => {
    render(<AppNavBar />);
  });
});

//   it("matches snapshot DOM node structure", () => {
//     const { asFragment } = render(
//       <Header onClose={mockToggleHeader} currentRecipe={currentRecipe} />
//     );
//     expect(asFragment()).toMatchSnapshot();
//   });
// });

// describe("Click Event", () => {
//   it("calls onClose handler", () => {
//     const { getByText } = render(
//       <Modal onClose={mockToggleHeader} currentRecipe={currentRecipe} />
//     );
//     fireEvent.click(getByText("Close this Header"));
//     expect(mockToggleHeader).toHaveBeenCalledTimes(1);
//   });
// })
