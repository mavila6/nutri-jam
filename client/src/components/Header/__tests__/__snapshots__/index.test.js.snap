exports[`Header component matches snapshot DOM node structures 1`] = `
<DocumentFragment>
  <div 
  class="headerBackdrop" 
  >
  <div 
  class="headerContainer"
  >
  <h3
  class="headerTitle">
  Caldo de Queso
  </h3>
  <img 
  alt="current recipe"
  src="1.jpg"
  />
  <p>
  Lorem ipsum dolor sit amet
  </p>
  <button 
  type="button" 
  >
  Close this Header
  </button>
  </div>
  </div>
  </DocumentFragment>
  `;
